flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/analyze/NonDeterministicUpdateAnalyzerTest.java

```diff
@@ -417,8 +486,8 @@ void testOverAggregateWithNonDeterminismInOrderBy() {
 
     @Test
     void testOverAggregateWithNonDeterminismInProjection() {
-        util.getTableEnv()
-                .getConfig()
+        var env = util.getTableEnv();
```

Comment:
Why can't we use `tEnv` here?

Reply:
ops

Reply:
We can, had missed on test in the refactoring. Updating.